region/crateria/central/Climb Supers Room.json

```diff
@@ -477,6 +490,7 @@
         "canLateralMidAirMorph",
         "canSpringFling"
       ],
+      "flashSuitChecked": true,
```

Comment:
It's possible `canLateralMidAirMorph` is `complexCarry` too since that is the most common place to fumble an input.  With the hard stuff, you are paying attention to your flash suit.

Reply:
I'm not seeing the risk with `canLateralMidAirMorph`. You'll be pressing down to morph, so it wouldn't be possible to accidentally press up?